styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -363,7 +365,9 @@ private String apiUrl(CharSequence... parts) {
   }
 
   private ByteString send(Request request) throws ExecutionException, InterruptedException {
-    final Response<ByteString> response = client.send(request).toCompletableFuture().get();
+    final Response<ByteString> response =
+        client.send(request.withHeader("Client-Version", STYX_CLI_VERSION)).toCompletableFuture()
```

Comment:
It looks at least Apollo doesn't touch ```User-Agent```, so use that instead of reinventing the wheel.